cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -314,6 +316,7 @@ protected void configure() {
       bind(ProgramLifecycleService.class).in(Scopes.SINGLETON);
       bind(NamespaceAdmin.class).to(DefaultNamespaceAdmin.class).in(Scopes.SINGLETON);
       bind(NamespaceQueryAdmin.class).to(DefaultNamespaceQueryAdmin.class).in(Scopes.SINGLETON);
+      bind(PreviewStore.class).to(DefaultPreviewStore.class);
```

Comment:
Adding the binding here will make cdap master also have this binding. It is better be in a separate Preview module so that we can choose where to use in (in 3.5, it should only be in StandaloneMain).